test/webhook-apicoverage/view/html_display.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package view
+
+import (
+	"fmt"
+	"strings"
+
+	"knative.dev/pkg/test/webhook-apicoverage/coveragecalculator"
+)
+
+// HTMLHeader sets up an HTML page with the right style format
+var HTMLHeader = fmt.Sprintf(`
```

Comment:
This should use https://godoc.org/html/template :(

Reply:
deferred